Corona-Warn-App/build.gradle

```diff
@@ -487,7 +487,7 @@ dependencies {
     implementation "com.airbnb.android:lottie:3.5.0"
 
     // HCert
-    implementation("com.upokecenter:cbor:4.5")
+    implementation("com.upokecenter:cbor:4.5.1")
```

Comment:
this is not the latest , can we upgrade to the latest version [v4.5.2](https://github.com/peteroupc/CBOR-Java/releases/tag/v4.5.2)

Reply:
latest version works fine too

Reply:
Sure!